src/microbe_stage/MicrobeAI.cs

```diff
@@ -370,34 +377,43 @@ private void RunAndTumble(Random random)
         microbe.State = Microbe.MicrobeState.Normal;
 
         var usefulCompounds = microbe.TotalAbsorbedCompounds.Where(x => microbe.Compounds.IsUseful(x.Key));
+        var compoundsPriority = ComputeCompoundsPriorities(usefulCompounds.ToList());
```

Comment:
Could we call this compoundWeights instead of compoundsPriority? Ultimately this is more of a weights-based algorithm than any kind of sort.

Reply:
compoundWeights makes more sense for the var here (because context) indeed, but I feel like the function name ComputeCompoundsPriorities would not describe well its computations. 

Changing one but not the other is setting me off a bit, I have mixed feelings over this one.

Reply:
Personally I feel like changing both names solves the problem. Take your time coming up with something you're happy with. I just read "priority" and thought "sorting", which isn't being done here.

Reply:
I get the confusion point here, I think you are right on the issue overall. I'll see what I can find after a good rest.